shared/src/business/entities/Message.ts

```diff
@@ -50,9 +51,10 @@ export class Message extends JoiValidationEntity {
     this.createdAt = rawMessage.createdAt || createISODateString();
     this.leadDocketNumber = rawMessage.leadDocketNumber;
     this.docketNumber = rawMessage.docketNumber;
-    this.docketNumberWithSuffix = `${rawMessage.docketNumber}${
-      rawMessage.docketNumberSuffix ? rawMessage.docketNumberSuffix : ''
-    }`;
+    this.docketNumberWithSuffix = Case.getDocketNumberWithSuffix({
+      docketNumber: rawMessage.docketNumber,
+      docketNumberSuffix: rawMessage.docketNumberSuffix,
+    });
```

Comment:
We store `docketNumber` and `docketNumberSuffix` in the DB. We could store `docketNumberWithSuffix` as a column in our DB rather than stitching the two pieces of data together using this function. However, we opted for the latter approach instead because 1) unless it is computationally expensive, it is good practice to persist only the data you need and 2) it avoids a developer seeing `docketNumber`, `docketNumberSuffix`, and `docketNumberWithSuffix` and wondering, "Wait, is there a difference between `docketNumber + docketNumberSuffix` and `docketNumberWithSuffix`? Will `docketNumberWithSuffix` always exist? Should I do something like `docketNumberWithSuffix || docketNumber + docketNumberSuffix` just to be safe?" and so on.